aws/rust-runtime/aws-types/src/lib.rs

```diff
@@ -15,11 +15,11 @@
 
 pub mod app_name;
 pub mod build_metadata;
-pub mod config;
 pub mod credentials;
 #[doc(hidden)]
 pub mod os_shim_internal;
 pub mod region;
+pub mod sdk_config;
```

Comment:
do you think we should just re-export SdkConfig into the root?

Reply:
the duplication required by `sdk_config::SdkConfig` seems a little bit cumersome